apps/eew/sceewlog/sceewdump.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env seiscomp-python
+"""
+Copyright (C) by ETHZ/SED
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+Author of the Software: Fred Massin
+"""
+
+from __future__ import absolute_import, division, print_function
```

Comment:
Not sure if this is needed still as the support for Python 2 has been dropped with SC 5.

Reply:
Damn, I took scevtstream 4.9.2 as a model to do this (https://github.com/SeisComP/main/blob/4.9.2/apps/python/scevtstreams.py) , but of course, there are parts that I didn't really take time to review, e.g.: the imports ... yes it is totally superfluous. Thanks.